check.go

```diff
@@ -57,6 +57,70 @@ func (d *DBFT[H]) checkCommit() {
 		return
 	}
 
+	// TODO: it should be considered: add PreCommit message instead of CommitAck and
+	// send this PreCommit *before* Commit. This will allow to create a completely
+	// custom message for keys exchange whereas the final Commit will keep only final
+	// signature bytes, as with usual dBFT.
+	if d.isAntiMEVExtensionEnabled() {
```

Comment:
My question is whether it only checks the signature and quantity of the commit message, but is it verified that the decrypted fragments of each transaction included in the commit can be decrypted normally?

Reply:
Both Commit and CommitAck messages need to be verified, and for their verification PreBlock and Block are required. Thus, signature fragments verification is implemented as Block’s and PreBlock’s Verify methods:
Commit's verification is here: https://github.com/nspcc-dev/dbft/blob/942dd05f871e437bfe159ca78d429d0732ef1d5c/dbft.go#L401

CommitAck's verifiation (final block BLS signatures verification) is here: https://github.com/nspcc-dev/dbft/blob/942dd05f871e437bfe159ca78d429d0732ef1d5c/dbft.go#L597

> is it verified that the decrypted fragments of each transaction included in the commit can be decrypted normally?

It'll be verified inside these `Verify` callbacks, and it will be implemented on the dBFT user side since dBFT is just an abstract library.